src/main/java/in/twizmwaz/cardinal/module/modules/teamPicker/TeamPicker.java

```diff
@@ -52,12 +55,20 @@ public Inventory getTeamPicker(Player player) {
                 totalPlayers += team.size();
             }
         }
```

Comment:
You shouldn't update their inventory, because they aren't switching teams. Maybe try getting the affected items and changing their lore and display name to the right ones. As @SungMatt said though, there isn't much reason to have a livestreamer rank.